types/mssql/index.d.ts

```diff
@@ -391,3 +391,6 @@ export declare class PreparedStatement extends events.EventEmitter {
 }
 
 export declare class PreparedStatementError extends MSSQLError {}
+
+export declare function connect(config: string | config | null | undefined, callback: (err?: any, pool?: ConnectionPool) => void): void;
+export declare function connect(config?: string | config): Promise<ConnectionPool>;
```

Comment:
Sorry for missing the updates.

Can we keep the `config` type consistent?

```suggestion
export declare function connect(config?: string | config, callback: (err?: any, pool?: ConnectionPool) => void): void;
export declare function connect(config?: string | config): Promise<ConnectionPool>;
```